openapi.yml

```diff
@@ -0,0 +1,189 @@
+          description: An array of activities
+      summary: Get list of activities for a team
+  "/teams/{teamId}/activities/{activityId}":
+    get:
+      parameters:
+        - name: teamId
+          description: Numeric ID of the team
+          schema:
+            type: integer
+          in: path
+          required: true
+        - name: activityId
+          description: Numeric ID of the activity
+          schema:
+            type: integer
+          in: path
+          required: true
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Activity"
+          description: A single activity detail
+      summary: Get a single activity detail
+  "/teams/{teamId}/members":
+    get:
+      parameters:
+        - name: teamId
+          description: Numeric ID of the team
+          schema:
+            type: integer
+          in: path
+          required: true
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Member"
+          description: An array of team members
+      summary: Get list of team members
+  "/teams/{teamId}/members/{memberId}":
+    get:
+      parameters:
+        - name: teamId
+          description: Numeric ID of the team
+          schema:
+            type: integer
+          in: path
+          required: true
+        - name: memberId
+          description: Numeric ID of the member
+          schema:
+            type: integer
+          in: path
+          required: true
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Member"
+          description: A single team member
+      summary: Get a single team member
+  "/teams/{teamId}/notes":
+    get:
+      parameters:
+        - name: teamId
+          description: Numeric ID of the team
+          schema:
+            type: integer
+          in: path
+          required: true
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Note"
+          description: An array of notes
+      summary: Get list of notes for the team
+components:
+  schemas:
+    Team:
+      type: object
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+        description:
+          type: string
+    Activity:
+      type: object
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+        description:
+          type: string
+        date:
+          format: date-time
+          type: string
+        location:
+          type: string
+    Member:
+      type: object
+      properties:

```

Comment:
The properties should correspond to the following data structure:

```
 { "address" : " ",
    "email" : "info@holdsport.dk",
    "id" : 78776,
    "mobile" : "12345678",
    "name" : "Demo Demo",
    "profile_picture_path": "", 
    "role" : 2
  },
  { "address" : "8000 Aarhus",
    "email" : "fran@juj.dk",
    "id" : 79413,
    "mobile" : "",
    "name" : "Kkk Kkkk"
    "profile_picture_path": "/media/BAhbB1sHOgZmSSI4MjAxMy8wg", 
    "role" : 1
  }
  ```

Reply:
done in latest commit